python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -117,9 +118,6 @@ def __init__(self,
         for local_path in self.config["file_mounts"].values():
             assert os.path.exists(local_path)
 
-        # List of resource bundles the user is requesting of the cluster.
-        self.resource_demand_vector = []
```

Comment:
Source of truth for `request_resources` was moved to load_metrics (where it arguably should've been all along), so we can report it and the other resource demands without an autoscaler.